helpers/io_utils.py

```diff
@@ -6,7 +6,8 @@
 from geopandas import GeoDataFrame
 from pandas import DataFrame as PandasDataFrame
 from pyspark.sql import DataFrame as SparkDataFrame
-from pyspark.sql.functions import col, date_trunc, to_timestamp
+from pyspark.sql.functions import col, date_trunc, to_timestamp, lit
```

Comment:
doesn't look like `lit` is used in this file

Reply:
oh weird, sorry i missed that! and definitely weird it didn't break anything before